packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -103,6 +109,35 @@ class MemoryController {
         .toList();
   }
 
+  void initializeLibraryFilters() {}
+
+  Future computeLibraries() async {
+    if (libraryCollection == null) {
+      final VM vm = await serviceManager.service.getVM();
+      // TODO(terry): Need to handle a possible Sentinel being returned.
+      final List<Isolate> isolates =
+          await Future.wait(vm.isolates.map((IsolateRef ref) async {
+        return await serviceManager.service.getIsolate(ref.id);
+      }));
+
+      libraryCollection = LibraryCollection(libraryFilters);
+      for (LibraryRef libraryRef in isolates[0].libraries) {
+        final Library theLibrary =
+            await serviceManager.service.getObject(_isolateId, libraryRef.id);
+        libraryCollection.addLibrary(theLibrary);
+      }
+
+      libraryCollection.computeDisplayClasses();
+    }
+  }
+
+  List<String> sortLibrariesByNormalizedNames() {
+    final List<String> normalizedNames =
+        libraryCollection.librarires.keys.toList();
+    normalizedNames.sort((a, b) => a.compareTo(b));
+    return normalizedNames;
+  }
+
   Future<dynamic> getObject(String objectRef) async =>
```

Comment:
why isn't this Future<bool> instead of Future<dynamic>

Reply:
No reason - done.